### DISCUSSION TITLE
Is it free?

### DISCUSSION THREAD

**Discussion by magnuscodus** [Da2dalus/CerebroSMS#8] [Q&A]
Hello, I have tried to analyze the Source Code with my minimal coding skills, and I've came to the point to ask the question of how do you deal with the paid textbelt api. I went on to their website and I've seen that it is a paid api. 

Any insights on this?

Maybe explain briefly the structure/logic of the programm? ^^ Cheers!

**Comment by Da2dalus** [comment]
I know it is a paid api. I have been searching for free alternatives. I found something called Twillio, but it also has a limit.
For now you can only use Textbelt. Use the api key "textbelt" to get one free sms a day.

In the next version I will add detailed comments.
